File: ascii-doc-validator/tests/code/java/StringUtils.java
package com.example;

public class StringUtils {

    /**
     * Возвращает строку с заглавной первой буквой.
     */
    public String capitalize(String input) {
        if (input == null || input.isEmpty()) return input;
        return input.substring(0,1).toUpperCase() + input.substring(1);
    }

    /**
     * Переворачивает строку задом наперёд.
     */
    public String reverse(String input) {
        return new StringBuilder(input).reverse().toString();
    }
}

File: ascii-doc-validator/tests/code/python/string_utils.py
# tests/code/python/string_utils.py
class StringUtils:
    """Утилитарные методы работы со строками."""

    @staticmethod
    def capitalize(input_str: str) -> str:
        if not input_str:
            return input_str
        return input_str[0].upper() + input_str[1:]

    @staticmethod
    def reverse(input_str: str) -> str:
        return input_str[::-1]